### DISCUSSION TITLE
Sorting catalogs by year with autocatalog

### DISCUSSION THREAD

**Discussion by smirnoffmv** [vuepress-theme-hope/vuepress-theme-hope#3859] [Q&A] [answered]
How do I sort catalogs by year so that 2024 is at the top and 2023 is at the bottom? Sorting using order does not work (it is only for articles)

Now:

![1](https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/90478289/46f387df-1f36-4915-a06d-64fec291e847)

**I want:**

![1](https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/90478289/7a2ee4bc-4ffd-49f0-add4-2bab95eb380f)

**Comment by Mister-Hope** [comment, marked as answer]
`dir.order` should work for directories I think. Double check that.

Besids, you are always free to do full customizations: https://plugin-auto-catalog.vuejs.press/config.html#defineautocataloggetter

**Reply by smirnoffmv** [reply]
Don't work :-( I write in Frontmatter `dir.order: -2024` on 2024 year and `dir.order: -2023` on 2023, but nothing.

![Скриншот 26-01-2024 182224](https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/90478289/3e12e865-929d-405b-b978-994bf19016bf)

**Reply by Mister-Hope** [reply]
learn some basics about a.b in yaml and json

**Comment by smirnoffmv** [comment]
I'm new to YAML, but for future users of the topic, to sort directories, you need to write

```yml
---
dir:
  order: 1 # the number is in order
---
```

**Reply by Mister-Hope** [reply]
we are planning to add a yaml guide to theme docs.
